tads/osportable.cc

```diff
@@ -577,9 +577,125 @@ osfmode( const char *fname, int follow_links, unsigned long *mode,
     return ok;
 }
 
+#ifdef _WIN32
+/*
+ *   Copied and adapted from tads2/osnoui.c
+ *   Get the DOS attributes for a file
+ */
+unsigned long oss_get_file_attrs(const char *fname)
```

Comment:
Since this function is not used outside of this file, it ought to be marked ``static``.